app/uk/gov/hmrc/helptosavereminder/repo/HtsReminderRepository.scala

```diff
@@ -201,13 +201,23 @@ class HtsReminderMongoRepository @Inject()(mongo: ReactiveMongoComponent)
       case _ => status.ok
     }
 
+  def statusCheck(status: WriteResult): Boolean =
```

Comment:
isn't that just:

status.n != 0

?

Reply:
Correct But cannot use due to wartremover.